tests/test_data/rhsvc/uac-w.json

```diff
@@ -1,6 +1,6 @@
 {
 	"uacHash": "54598f02da027026a584fd0bc7176de55a3e6472f4b3c74f68d0ae7be206e17c",
-	"active": "True",
+	"active": true,
```

Comment:
Is this correct? I know that RM are providing all booleans as strings - is RHSvc definitely passing them to RHUI as booleans?

Reply:
You are correct. I've updated the code.